beacon-chain/powchain/prometheus.go

```diff
@@ -0,0 +1,153 @@
+package powchain
+
+import (
+	"context"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prysmaticlabs/prysm/shared/clientstats"
+)
+
+type BeaconNodeStatsUpdater interface {
+	Update(stats clientstats.BeaconNodeStats)
+}
+
+type PowchainCollector struct {
```

Comment:
I didnt dive too far into this, do `PowchainCollector` and `Sync...` fields all need to be exported?

Reply:
It's exported so that beacon-node/node can pass it into NewService and so that tests can use a mock.